modules/lang-painless/src/main/java/org/elasticsearch/painless/ir/ClassNode.java

```diff
@@ -24,8 +24,8 @@
 import org.elasticsearch.painless.MethodWriter;
 import org.elasticsearch.painless.ScriptClassInfo;
 import org.elasticsearch.painless.phase.IRTreeVisitor;
-import org.elasticsearch.painless.symbol.WriteScope;
```

Comment:
How did this pass style checking?

Reply:
I don't think we check order. I "optimized" imports for all the files in Intellij so hopefully it picks this up.